x/evm/statedb/journal.go

```diff
@@ -139,6 +141,10 @@ type (
 		address *common.Address
 		slot    *common.Hash
 	}
+	precompileChange struct {
+		ms sdk.MultiStore
```

Comment:
[Question] Is there any reason why multiStore was changed to ms and events to es?